fabric/build.gradle

```diff
@@ -77,11 +77,6 @@ dependencies {
     modRuntimeOnly "me.djtheredstoner:DevAuth-fabric:${devauth_version}"
 
     testImplementation("net.fabricmc:fabric-loader-junit:${fabric_loader_version}")
-    testImplementation("org.junit.jupiter:junit-jupiter:${junit_version}")
-}
-
-test {
-    useJUnitPlatform()
```

Comment:
Why do you think you should change how our testing works? We use fabric for JUnit, so we can test with game classes as well.

Reply:
I don't know in my head it made more sense for the tests to be in common but now I'm not so sure...

Reply:
In regular projects yes. This is a game. We can't JUnit all 3 projects, that leaves fabric the best option.

Reply:
moved it back